template/src/index.html.js

```diff
@@ -0,0 +1,26 @@
+import stylesheet from '@kaliber/build/lib/stylesheet'
+import javascript from '@kaliber/build/lib/javascript'
+import styles from '/index.css'
+
+Index.routes = {
+  // match :: ({ pathname: String, search: String }, Request) -> { status: Number, data: Object }
+  match: (location, request) => ({ status: 200, data: {} })
+}
+
+export default function Index ({ location, data }) {
```

Comment:
I'm in doubt. Should we advocate the use of the one with the router, or the static version with the `500.html.js` and `400.html.js` templates?

Reply:
As discussed: we'll go for the static version.